lib/rules/prefer-find-by.ts

```diff
@@ -0,0 +1,88 @@
+import { ESLintUtils, TSESTree } from '@typescript-eslint/experimental-utils';
+import {
+  isIdentifier,
+  isCallExpression,
+  isMemberExpression,
+  isArrowFunctionExpression,
+} from '../node-utils';
+import { getDocsUrl, SYNC_QUERIES_COMBINATIONS } from '../utils';
+
+export const RULE_NAME = 'prefer-find-by';
+
+type Options = [];
+export type MessageIds = 'preferFindBy';
+// TODO check if this should be under utils.ts - there are some async utils
```

Comment:
We have `ASYNC_UTILS` in `utils.ts` but it includes more methods than here.

Reply:
Maybe it's a good idea to keep all different async utils under an enumerate type and then build `ASYNC_UTILS` all the enumerate options?

Reply:
after giving some though, I am not sure of moving this one out. It's not a list of all `wait` methods, but just a list of the ones this rule cares about. Not sure how useful it could be outside of here, as it's not _all_ the `wait` methods. Either way, we could centralize them in the future if we see the chance